### DISCUSSION TITLE
Can I use github pages blog as my personal blog (not related to development, programming, etc)?

### DISCUSSION THREAD

**Discussion by good-nice** [community/community#21710] [Other Feature Feedback, Questions, & Ideas]
<p>Hello dear community members.</p>
<p>I have questions.</p>
<ol>
<li>Can I use github pages blog as my personal blog (not related to development, programming, applications) for example “cat blog”?</li>
<li>What are the limits except: “GitHub Pages source repositories have a recommended limit of 1GB . Published GitHub Pages sites may be no larger than 1 GB. GitHub Pages sites have a soft bandwidth limit of 100GB per month. GitHub Pages sites have a soft limit of 10 buildings per hour.”?</li>
</ol>

**Comment by RDIL** [comment, marked as answer]
<p>Hey!</p>
<p>Yes, you can use GitHub pages to host a personal blog.  I believe GitHub won’t mind.  </p>
<p>There are some basic limits in addition to the ones listed above:</p>
<p>* If you are using Jekyll/Markdown, you can’t use non-whitelisted gems/plugins/themes</p>
<p>* Unless you have the pro plan, you will need to have the repository public</p>
<p>If you have any other questions, feel free to respond!</p>
